# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 228200dcaf93
# Full Hash: 228200dcaf93bb3f10e338157325ef6227e8a0a5
# Author: Kelsey Gilbert <kelsey.gilbert@mozilla.com>
# Date: 2023-02-15 18:16:35
# Regressor Bug: 1799258
# File Overlap Count: 1
# Description:
#   Bug 1799258 - Do color-management on Windows+DComp via IDCompositionFilterEffects. r=sotaro
#   
#   + Add gfx.color_management.rec709_gamma_as_srgb:true. :'(
#   
#   In particular, rec709(16/255) -> srgb(31/255). Even though it's
# ==============================================================================

diff -r b25110652394 -r 228200dcaf93 modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Wed Feb 15 08:29:41 2023 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Wed Feb 15 08:29:41 2023 +0000
@@ -5775,6 +5775,11 @@
 #endif
   mirror: always
 
+- name: gfx.color_management.display_profile
+  type: DataMutexString
+  value: ""
+  mirror: always # But be warned: We cache the result.
+
 - name: gfx.color_management.force_srgb
   type: RelaxedAtomicBool
   value: false
@@ -5782,7 +5787,7 @@
 
 - name: gfx.color_management.native_srgb
   type: RelaxedAtomicBool
-#if defined(XP_MACOSX)
+#if defined(XP_MACOSX) || defined(XP_WIN)
   value: true
 #else
   value: false
@@ -5807,6 +5812,16 @@
   value: 0
   mirror: always
 
+- name: gfx.color_management.rec709_gamma_as_srgb
+  type: RelaxedAtomicBool
+  value: true # Tragic backwards compat.
+  mirror: always
+
+- name: gfx.color_management.rec2020_gamma_as_rec709
+  type: RelaxedAtomicBool
+  value: true # Match naive behavior, but hopefully we can stop soon!
+  mirror: always
+
 - name: gfx.compositor.clearstate
   type: RelaxedAtomicBool
   value: false